nautobot/core/testing/mixins.py

```diff
@@ -150,14 +149,13 @@ def assertHttpStatus(self, response, expected_status, msg=None):
         if isinstance(expected_status, int):
             expected_status = [expected_status]
         if response.status_code not in expected_status:
+            err_message = f"Expected HTTP status(es) {expected_status}; received {response.status_code}:"
             if hasattr(response, "data"):
                 # REST API response; pass the response data through directly
-                err = response.data
-            else:
-                # Attempt to extract form validation errors from the response HTML
-                form_errors = testing.extract_form_failures(response.content.decode(response.charset))
-                err = form_errors or response.content.decode(response.charset) or "No data"
-            err_message = f"Expected HTTP status(es) {expected_status}; received {response.status_code}: {err}"
+                err_message += f"\n{response.data}"
+            # Attempt to extract form validation errors from the response HTML
+            form_errors = testing.extract_form_failures(response.content.decode(response.charset))
+            err_message += "\n" + str(form_errors or response.content.decode(response.charset) or "No data")
```

Comment:
In troubleshooting some test failures, I found that for `NautobotUIViewSet`, the response includes a REST-style `data` but also has information in `response.content` - so I changed this from an if/else case to a cumulative construction of the `err_message`.